docs/dev_review.rst

```diff
@@ -1,51 +1,56 @@
-.. libCellML Review Process::
+.. _dev_review:
 
 ==============
-Review Process
+Review process
 ==============
 
-Check for the Green Tick
-========================
+This chapter details the responsibilities of the repository owners, and will help contributors to understand how their code contributions will be assessed and reviewed.
 
-Before accepting a tranche of work into the libCellML prime repository check that `Buildbot <https://buildbot.net/>`_ has tested and passed the code.
+Check for the green tick
+========================
+Before accepting a tranche of work into the libCellML prime repository check that the :libcellml_buildbot:`libCellML Buildbot<>` has tested and passed the code.
 The status of the code is shown in the last commit of a pushed group of commits in the pull request.
 The last commit will have a red cross for a failed build or a green tick for a passed build.
 Obviously, make sure that the last commit has a green tick before merging.
 
-Read the Documentation
+The "Details" link at the bottom of the pull request web page will take you to the build records of the library.
+This page shows the results of the unit tests for each target operating system, the results of the coverage test, the results of the memory check test, and the results of the documentation build.
+
+The coverage statistics for the library (generated from :gcovr:`gcovr<>` ) can be reached through the :ref:`Coverage Statistics <coverage_statistics>` page **TODO** is this still true??
+
+Read the documentation
 ======================
 
+**TODO** Not sure any of this is current??
```

Comment:
It is.

Reply:
Then it doesn't make sense to me?  Here's what the link is pointing to: 
![image](https://user-images.githubusercontent.com/46887220/107562511-cb573c80-6c44-11eb-9e91-11d8ca73a85b.png)
What does this refer to? 
>> The Documentation Builder link (entry 2 in step 5) will take you to the build for the documentation.
On this page, you can see the steps taken to build the documentation.
In the last step of the build (step 7), there is a link 'dox' (entry 2) that will take you to the built documentation.